app/timetables/tests/test_utils.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
+
+from app.timetables.utils import timestamp_seconds
+
+
+class UtilsTest(TestCase):
+    """Test utils module."""
+
+    def test_timestamp_seconds(self):
```

Comment:
@mfalade: Sure, it's impure. However, it works as intended. Within the problem domain of getting unique response each time it's called, I can think of only one single condition for testing purpose: getting a unique integer per call per application instance. In other words, at no point in time, during runtime, should the function return duplicate integer. And using `set` collection datatype to gather all the responses of the function validates the uniqueness of the responses on a runtime basis.

I initially tried using random module, but prevention of re-occurrence of outcomes is not guaranteed as it's based on probability. As you suggested implementing a pure function, do you any implementation idea in mind?